### PR TITLE
Updated CLI, Added Generate Bulk Testing Content, Fixed bugs reported by NCHS

### PR DESCRIPTION
**Pull request by saulakravitz**
- Added CLI entry to generate bulk testing content based on the 3 connectathon records.

generatedrecords startcert numcerts out_dir state

- Updated README
- Added 'help' command that describes all commands. 
- Changed response with zero arguments to be the same as "help"
- Added command "fakerecord" which was previous default response.
- Added some tests to prevent empty junk records to be added to a deathrecord, as per NVSS-399.
- fixed bugs reported by NCHS
- added support for StatusMessage

### CODE REVIEW

File: VRDR.CLI/Program.cs [hunk 1]
```diff
@@ -20,6 +21,7 @@ class Program
     {
         static int Main(string[] args)
         {
+            Console.WriteLine($"{args.Length}");
```
Comment (smacadam) [inline VRDR.CLI/Program.cs:24]: looks like this was for testing, should this be removed?

**Comment by smacadam** [comment]
Tested the CLI and it looks good, a few comments to remove console logs and add an example to the README

**Comment by smacadam** [comment]
The test says its failing because it can't find the Status Message but I see it in the test .csproj file, I'm not sure why it's failing to find it...

**Comment by saulakravitz** [comment]
> 

@smacadam  Oops... didn't realize tests weren't running....now they should run successfully.

**Review by smacadam — APPROVED** [review]

File: VRDR.CLI/Program.cs [hunk 2]
```diff
@@ -812,6 +821,39 @@ record = coding.DeathRecord;
...
+                    Console.WriteLine("Must supply a starting certificate number greater than 0");
+                    return(1);
+                }else if (String.IsNullOrWhiteSpace(args[3]) || !Directory.Exists(args[3])){
+                    Console.WriteLine("Must supply a valid output directory");
+                    return(1);
+                }else if (count <= 0 ){
+                    Console.WriteLine("Must supply a count greater than 0");
+                    return(1);
+                }
+                Console.WriteLine(" count = ${count} output_directory = ${output_directory}");
```
[the remark this answers is not in the document]
Comment (saulakravitz) [reply VRDR.CLI/Program.cs:878]: deleted

File: VRDR.CLI/Program.cs [hunk 3]
```diff
@@ -812,6 +821,39 @@ record = coding.DeathRecord;
...
                 return 0;
             }
+            else if (args.Length == 5 && args[0] == "generatedrecords")
+            {
+                Console.WriteLine("generatedrecords");
+                int start_certificate_number = Int16.Parse(args[1]);
+                int count = Int16.Parse(args[2]);
+                string output_directory = args[3];
+                string state = args[4];
+                if (start_certificate_number < 0) {
```
[the remark this answers is not in the document]
Comment (saulakravitz) [reply VRDR.CLI/Program.cs:868]: done

**Pull request merged**